styleguide/components/41-organisms/search-block/_search-block.scss

```diff
@@ -0,0 +1,71 @@
+.search-block {
+  @include theme('background-color', 'color-primary--lighten-4', 'search-block-background');
+
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  margin-top: 3.1rem;
+  padding: 0 .8rem 2rem;
+
+  @include tablet {
+    flex-wrap: nowrap;
+    justify-content: flex-start;
+    margin-top: 0;
+    padding: 2rem 3.7rem 2rem ($gutter-width*2);
+  }
+
+  h2 {
+    @extend %h4;
+  }
+
+  ul {
+    margin: 0 0 2rem;
+    list-style-type: none;
+
+    @include tablet {
+      margin-left: auto;
+    }
+
+    li {
+      position: relative;
+      margin-bottom: .5rem;
+      padding-left: 1.8rem;
+      line-height: 1.8;
+
+      &,
+      > * {
+        font-size: .8rem;
+      }
+    }
+
+    i {
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      left: 0;
+      height: 2rem;
```

Comment:
Don't mix em and rem
font-size is em, so height should be too.

Reply:
Fixed.